src/visitors/setInstructionAccountDefaultValuesVisitor.ts

```diff
@@ -165,6 +165,14 @@ export const DEFAULT_INSTRUCTION_ACCOUNT_DEFAULT_RULES: InstructionAccountDefaul
       ),
       ignoreIfOptional: true,
     },
+    {
+      account: /^coreProgram|mplCoreProgram$/,
```

Comment:
Because `core` is a very generic term — unlike `candyMachine` for instance — I'd rather we only support the `mplCoreProgram` regex to identify this program.

```suggestion
      account: /^mplCoreProgram$/,
```

Reply:
Done. https://github.com/metaplex-foundation/kinobi/pull/171/commits/345700dc2ad802004cb895ea744f1a81e5fa11b8